### PR TITLE
Basic config

### PR DESCRIPTION
**Pull request by kamilsobik**
#Basic config

- [x] create new app
- [x] create setup
- [x] make page with logo

https://github.com/kamilsobik/codenews.dev/assets/99078349/6cfc21a7-0879-41fc-b894-a0802b04c8b3

### CODE REVIEW

File: .vscode/settings.json
Comment (piotr-pajak) [inline .vscode/settings.json:1]: **Suggest** How about adding .`vscode` to .gitignore? 
In my case i don't need this file. 
Same as WebStrom generates `.workspace.xml` which you don't need.

**Review by piotr-pajak — COMMENTED** [review]
Please format all files in project with prettier :pray:

File: package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+		"start": "next start",
+		"lint": "next lint --fix --dir src",
+		"lint:ci": "next lint --dir src",
+		"prepare": "husky install"
+	},
+	"dependencies": {
+		"next": "^13.4.12",
+		"react": "^18.2.0",
+		"react-dom": "^18.2.0",
+		"react-icons": "4.8.0",
```
[the remark this answers is not in the document]
Comment (kamilsobik) [reply package.json:17]: Not now, I delete it from project

File: .vscode/settings.json
Comment (kamilsobik) [reply .vscode/settings.json:1]: Add this file

**Review by piotr-pajak — APPROVED** [review]

**Pull request merged**